SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Reporting Security Issues
+
+If you discover a security issue in stylelint, please report it by sending an
+email to .
```

Comment:
/cc @stylelint/core we need email here :smile:

Reply:
Shall we just add `stylelint@gmail.com` for now to get this merged?

Reply:
Yeah. Is anyone checking this email? :)

Reply:
I do :)